packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -59,7 +59,10 @@ export class ComponentUtils {
       if (!isNullOrUndefined(cmpArray)) {
         cmpArray = cmpArray instanceof Array ? cmpArray : [cmpArray];
         for (const cmp of cmpArray) {
-          if (!isNullOrUndefined(cmp.fullName)) {
+          if (
+            !isNullOrUndefined(cmp.fullName) &&
+            cmp.manageableState === 'unmanaged'
```

Comment:
Eventually we'll move this to a set and include the enum value for unlocked packaging